src/parser/WASMParser.cpp

```diff
@@ -1399,11 +1423,22 @@ class WASMBinaryReader : public wabt::WASMBinaryReaderDelegate {
         ASSERT(peekVMStackValueType() == Walrus::Value::Type::I32);
         auto stackPos = popVMStack();
 
+        bool unify = isLastByteCodeEqz() && m_currentFunction->peekByteCode<Walrus::UnaryOperation>(m_lastEqzPos)->dstOffset() == stackPos;
```

Comment:
I would call this `isInverted`. In all other cases as well.

Reply:
Okay.